backend/clubs/views.py

```diff
@@ -233,6 +243,20 @@ def get_filename(self):
             return "{}.xlsx".format(slugify(name))
         return super().get_filename()
 
+    def partial_update(self, request, *args, **kwargs):
+        if request.data.get("accepted", False) is True and not request.user.has_perm(
+            "approve_club"
+        ):
+            return HttpResponseForbidden()
```

Comment:
We want to use [`PermissionDenied`](https://stackoverflow.com/a/50076664) instead of `HttpResponseForbidden` here. `HttpResponseForbidden` is used in Django, while `raise PermissionDenied` is used in DRF.